server/controllers/importers/graphml.js

```diff
@@ -208,5 +208,10 @@ module.exports = function (graphml) {
         });
     }
 
-    return semanticChecker(network);
+
+  if (network.errors.length === 0) {
+      return semanticChecker(network);
+  }
+  return network;
+
```

Comment:
More compact, and clearer on the intent that we are returning something for sure:

    return (network.errors.length === 0) ? semanticChecker(network) : network;